.github/workflows/conda-package.yml

```diff
@@ -124,7 +124,7 @@ jobs:
           OVERRIDE_INTEL_IPO: 1   # IPO requires more resources that GH actions VM provides
         run: |
           conda activate
-          conda build --no-test --python ${{ matrix.python }} --numpy 2 -c conda-forge --override-channels conda-recipe
+          conda build --no-test --python ${{ matrix.python }} --numpy 2.0 -c ${{ env.INTEL_CHANNEL }} -c conda-forge --override-channels conda-recipe
```

Comment:
Would it make sense to add a TODO comment noting that the changes need to be reverted once 2025.1 compiler is available of `conda-forge` channel?

Reply:
Added